OpenRA.Mods.Common/Traits/Carryall.cs

```diff
@@ -318,9 +318,11 @@ IEnumerable<IOrderTargeter> IIssueOrder.Orders
 			{
 				if (IsTraitDisabled)
 					yield break;
-
+				var orderPriority = 1;
```

Comment:
Please keep the empty line:
```suggestion

				var orderPriority = 1;
```